images/macos/provision/core/xcode.ps1

```diff
@@ -52,6 +52,13 @@ $xcodeVersions | ForEach-Object {
     Rebuild-XcodeLaunchServicesDb -Version $_.link
 }
 
+# 14.0 and higher doesn't ship tvOS and watchOS, so installing it manually
+$xcodeVersions | Where-Object { $_.link -ge "14.0" } | ForEach-Object {
```

Comment:
I think we shouldn't use `$_.link -ge "14.0" because this way we compare strings instead of versions.
May be `{ $_.link.StartsWith("14.") } like https://github.com/actions/runner-images/blob/7a7fafda99ef4ddb3539fc19596dd1cfc1ab0edc/images/macos/helpers/Xcode.Installer.psm1#L154

Reply:
why not